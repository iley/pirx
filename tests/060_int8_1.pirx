extern func main(): int {
  var x: int8;

  x = 42i8;
  printf("%d\n", x);

  x = -42i8;
  printf("%d\n", x);

  x = 127i8;
  x = x + 1i8;
  printf("%d\n", x);

  return 0;
}
