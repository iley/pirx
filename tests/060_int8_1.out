42
-42
-128
